### DISCUSSION TITLE
Is it possible to figure out if someone is cheating by looking at the logs?

### DISCUSSION THREAD

**Discussion by Simpuhl** [lloesche/valheim-server-docker#190] [Q&A]
Somehow someone spawned like 10 bosses. I rolled back to the backup but I am trying to figure out if there is a way to see who did that... looking through the logs and I cannot find anything

**Comment by lloesche** [comment]
I don't know, but do you still have the complete logs? I'd be interested in looking at them. I wasn't even aware that anyone but the server is able to spawn anything. You'd think they restrict that server side.

**Comment by schildbach** [comment]
I think anyone can spawn bosses. For example, we fought 2 Eikthyrs by sacrificing the double amount of deer trophies at his altar.

**Comment by Simpuhl** [comment]
But all at the same time?

Get Outlook for Android<https://aka.ms/AAb9ysg>

________________________________

**Reply by rileydakota** [reply]
Can confirm - the game will let you simultaneously spawn as many of the same boss as you want provided you have the required material. Unless someone is intentionally spawning 20+ bosses in an attempt to lag the server, might not be worth looking at.
